resources/js/features/game-list/utils/column-definitions/buildPlayerGameProgressColumnDef.tsx

```diff
@@ -0,0 +1,33 @@
+import type { ColumnDef } from '@tanstack/react-table';
+import type { RouteName } from 'ziggy-js';
+
+import { PlayerGameProgressBar } from '@/common/components/PlayerGameProgressBar';
+
+import { DataTableColumnHeader } from '../../components/DataTableColumnHeader';
+
+interface BuildPlayerGameProgressColumnDefProps {
+  tableApiRouteName?: RouteName;
+}
+
+export function buildPlayerGameProgressColumnDef({
+  tableApiRouteName = 'api.game.index',
+}: BuildPlayerGameProgressColumnDefProps): ColumnDef<App.Platform.Data.GameListEntry> {
+  return {
+    id: 'progress',
+    accessorKey: 'game',
+    meta: { label: 'Progress', align: 'left' },
+    header: ({ column, table }) => (
+      <DataTableColumnHeader
+        column={column}
+        table={table}
+        sortType="quantity"
+        tableApiRouteName={tableApiRouteName}
+      />
+    ),
+    cell: ({ row }) => {
+      const { game, playerGame } = row.original;
+
+      return <PlayerGameProgressBar game={game} playerGame={playerGame} />;
+    },
```

Comment:
Noticed on this grid, but it happens on the Want to Play grid as well: The progress widget changes the cursor to a hand, making it seem like it should be clickable, but clicking does nothing.

Reply:
Agreed. In latest, if the tooltip can appear, the progress bar now links to the game page.